src/app/layout.tsx

```diff
@@ -1,34 +1,21 @@
-import type { Metadata } from "next";
-import { Geist, Geist_Mono } from "next/font/google";
-import "./globals.css";
+import { ReactNode } from 'react';
+import { metadata } from './memos/page';
+import Link from 'next/link';
 
-const geistSans = Geist({
-	variable: "--font-geist-sans",
-	subsets: ["latin"],
-});
-
-const geistMono = Geist_Mono({
-	variable: "--font-geist-mono",
-	subsets: ["latin"],
-});
-
-export const metadata: Metadata = {
-	title: "Create Next App",
-	description: "Generated by create next app",
-};
+//ReactNode であることを型定義する！
```

Comment:
コメントの`//`とコメント内容の間にはスペースを設けるのが一般的です
```suggestion
// ReactNode であることを型定義する！
```